### PR TITLE
adding Davies and Comrie 1985

### PR DESCRIPTION
**Pull request by SimonGreenhill**
A linguistic survey of Upper Yuat (another TNG.org list).

Some questions:
1. This list has verbs as "he bites" which I've mapped to "BITE". Is this ok?

2. "he gives it to me"  is mapped to "GIVE".  Should this be given its own new concept (give transitive?)

3. "axe (stone)" -> I've put this in AXE but perhaps could be a new concept set (STONE AXE)?

### CODE REVIEW

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -215,3 +215,4 @@ Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon
...
 Pagel-2018-200	Pagel, Mark AND Meade, Andrew	2018		200	basic	English	Global	http://rstb.royalsocietypublishing.org/highwire/filestream/38234/field_highwire_adjunct_files/0/rstb20160517supp1.pdf	Pagel2018		The list is based on the [210-item list of ABVD](:ref:Blust-2008-210), the  [207-item list of IELex](:ref:Dunn-2012-207), and the [100-item list by Grollemund et al.](:ref:Grollemund-2015-100). The RATE column describes the lexical replacement rate of an item per year, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund et al. (2015)](:bib:Grollemund2015), whereas, to the best of our knowledge, the list only contains 100 meanings.		
+Davies-1985-116	Davies, John AND Comrie, Bernard	1985		116	basic,areal	English	New Guinea	http://sealang.net/archives/pl/pdf/PL-A63.275.pdf	Davies1985		A 116 item wordlist survey of the Upper Yuat region. SIL survey list varient
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: I don't understand the last half-sentence. And could you add a full stop after the note ;)

File: concepticondata/conceptlists/Davies-1985-116.tsv [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Davies-1985-116-7	7	belly	1251	BELLY
+Davies-1985-116-8	8	skin	763	SKIN
+Davies-1985-116-9	9	man	1554	MAN
+Davies-1985-116-10	10	woman	962	WOMAN
+Davies-1985-116-11	11	bird	937	BIRD
+Davies-1985-116-12	12	dog	2009	DOG
+Davies-1985-116-13	13	he bites	1403	BITE
+Davies-1985-116-14	14	he sits	1416	SIT
+Davies-1985-116-15	15	he stands	1442	STAND
+Davies-1985-116-16	16	road	2457	PATH OR ROAD
```
Comment (LinguList) [inline concepticondata/conceptlists/Davies-1985-116.tsv]: Maybe use 667 ROAD, as PATH OR ROAD is a bit broader?

File: concepticondata/conceptlists/Davies-1985-116.tsv [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+Davies-1985-116-50	50	his thumb	1781	THUMB
+Davies-1985-116-51	51	his leg	1297	LEG
+Davies-1985-116-52	52	his heart	1223	HEART
+Davies-1985-116-53	53	his bone	1394	BONE
+Davies-1985-116-54	54	his blood	946	BLOOD
+Davies-1985-116-55	55	wing	1257	WING
+Davies-1985-116-56	56	tail	1220	TAIL
+Davies-1985-116-57	57	his father	1217	FATHER
+Davies-1985-116-58	58	his mother	1216	MOTHER
+Davies-1985-116-59	59	brother (older, of man)	1262	BROTHER
```
Comment (LinguList) [inline concepticondata/conceptlists/Davies-1985-116.tsv]: use 2414 OLDER BROTHER (OF MAN)

File: concepticondata/conceptlists/Davies-1985-116.tsv [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+Davies-1985-116-96	96	wet	1726	WET
+Davies-1985-116-97	97	full	1429	FULL
+Davies-1985-116-98	98	not	1240	NOT
+Davies-1985-116-99	99	three	492	THREE
+Davies-1985-116-100	100	four	1500	FOUR
+Davies-1985-116-101	101	no	1269	NO
+Davies-1985-116-102	102	he says	1458	SAY
+Davies-1985-116-103	103	he hears	1408	HEAR
+Davies-1985-116-104	104	he dies	1494	DIE
+Davies-1985-116-105	105	it burns	2102	BURN
```
Comment (LinguList) [inline concepticondata/conceptlists/Davies-1985-116.tsv]: use 1428 BURNING, as this is obviously intransitive

**Comment by xrotwang** [comment]
Should this go into v1.1?

**Comment by LinguList** [comment]
>    This list has verbs as "he bites" which I've mapped to "BITE". Is this ok?

I think it is okay. Obviously, for these languages, the elicitation gloss was chosen similar to that TNG list we used for the 2017 cognate detection paper (I keep forgetting the name), so 3rd person singular is similar to asking for the infinitive in Europe.

>    "he gives it to me" is mapped to "GIVE". Should this be given its own new concept (give transitive?)

I'd say we tolerate it for now. People working on the data in concrete should, however, make sure that the entry is multi-word or not (it happens in Chinese questionnaires, where the elicitation gloss is a sentence, but they only give one word, so if you KNOW what the people answered on these questions, it is even better, and you can be more explicit).

>    "axe (stone)" -> I've put this in AXE but perhaps could be a new concept set (STONE AXE)?

Have you grepped, whether this was already encountered? I think some lists may have STONE AXE, and they were either unlinked or linked to AXE, but if not, we may link it to AXE for the moment.

**Comment by LinguList** [comment]
> Should this go into v1.1?

we have 216 lists by now. if we have 217, we have an odd number, and I don't like odd numbers. The alternative would be to wait until we have 220 lists, but this may take another two weeks (@chrzyki is also looking into this). There are a few very small lists, so we could scale up to 220 today, but maybe it's better to work on the lingpy tutorial for the time being. As I know my dislike of odd numbers is not making any sense, I'll leave it to @SimonGreenhill to decide...

**Comment by SimonGreenhill** [comment]
how about I add another one so we get to 220?

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists.tsv]: done

File: concepticondata/conceptlists/Davies-1985-116.tsv (hunk 2 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Davies-1985-116.tsv]: done.

File: concepticondata/conceptlists/Davies-1985-116.tsv (hunk 3 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Davies-1985-116.tsv]: done

File: concepticondata/conceptlists/Davies-1985-116.tsv (hunk 4 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Davies-1985-116.tsv]: done

**Comment by SimonGreenhill** [comment]
Re AXE -- I know some New Guinea languages distinguish stone from iron axes, but we've only got 'iron axe' in Z'graggen 1980 (linked to AXE). I'll keep an eye on this and split in future if needed.

File: concepticondata/conceptlists.tsv [hunk 5]
```diff
@@ -215,3 +215,4 @@ Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon
...
 Pagel-2018-200	Pagel, Mark AND Meade, Andrew	2018		200	basic	English	Global	http://rstb.royalsocietypublishing.org/highwire/filestream/38234/field_highwire_adjunct_files/0/rstb20160517supp1.pdf	Pagel2018		The list is based on the [210-item list of ABVD](:ref:Blust-2008-210), the  [207-item list of IELex](:ref:Dunn-2012-207), and the [100-item list by Grollemund et al.](:ref:Grollemund-2015-100). The RATE column describes the lexical replacement rate of an item per year, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund et al. (2015)](:bib:Grollemund2015), whereas, to the best of our knowledge, the list only contains 100 meanings.		
+Davies-1985-116	Davies, John AND Comrie, Bernard	1985		116	basic,areal	English	New Guinea	http://sealang.net/archives/pl/pdf/PL-A63.275.pdf	Davies1985		A 116 item wordlist survey of the Upper Yuat region. A SIL survey list varient.
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv]: sorry, I know I'm a pain in the neck, but what does "varient" mean? Would: 

> A 116 item wordlist survey of the Upper Yuat region, based on a variant of survey lists by the summer institutes of linguistics (currently not in Concepticon yet).

that work for you?

**Comment by chrzyki** [comment]
@LinguList I could add Kibrik-2012-122 and Perlman-2018-30 on Friday, if that helps with any release decisions.

File: concepticondata/conceptlists.tsv (hunk 5 above)
Comment (xrotwang) [reply concepticondata/conceptlists.tsv]: @SimonGreenhill What do you think? Take @LinguList 's proposal and push v1.1 out the door?

**Comment by SimonGreenhill** [comment]
Yes -- sorry, wasn't trying to delay release, I'm just working through my 'to-do' list.

**Pull request merged**